public/apps/account/test/account-nav-button.test.tsx

```diff
@@ -13,92 +13,121 @@
  *   permissions and limitations under the License.
  */
 
-import { shallow } from 'enzyme';
-import React from 'react';
-import { AccountNavButton } from '../account-nav-button';
-import { getShouldShowTenantPopup, setShouldShowTenantPopup } from '../../../utils/storage-utils';
-
-jest.mock('../../../utils/storage-utils', () => ({
-  getShouldShowTenantPopup: jest.fn(),
-  setShouldShowTenantPopup: jest.fn(),
-}));
-
-describe('Account navigation button', () => {
-  const mockCoreStart = {
-    http: 1,
-  };
-
-  const config = {
-    multitenancy: {
-      enabled: 'true',
-      tenants: {
-        enable_private: 'true',
-        enable_global: 'true',
-      },
-    },
-    auth: {
-      type: 'dummy',
-    },
-  };
-
-  const userName = 'user1';
-
-  let component;
-  const setState = jest.fn();
-  const useStateSpy = jest.spyOn(React, 'useState');
-
-  beforeEach(() => {
-    useStateSpy.mockImplementation((init) => [init, setState]);
-    component = shallow(
-      <AccountNavButton
-        coreStart={mockCoreStart}
-        isInternalUser={true}
-        username={userName}
-        tenant="tenant1"
-        config={config as any}
-      />
-    );
-  });
-
-  afterEach(() => {
-    jest.clearAllMocks();
-  });
-
-  it('renders', () => {
-    expect(component).toMatchSnapshot();
-  });
-
-  it('should set modal when show popup is true', () => {
-    (getShouldShowTenantPopup as jest.Mock).mockReturnValueOnce(true);
-    shallow(
-      <AccountNavButton
-        coreStart={mockCoreStart}
-        isInternalUser={true}
-        username={userName}
-        tenant="tenant1"
-        config={config as any}
+exports[`Account navigation button renders 1`] = `
```

Comment:
I'm not sure if you added this change. But this test file has been changed to a snapshot file. Do you know why is that?